concourse-production.yml

```diff
@@ -28,3 +28,10 @@ instance_groups:
   azs: [z1]
   networks:
   - name: production-concourse
+- name: iaas-worker
+  vm_type: production-concourse-worker
+  vm_extensions: [production-concourse-iaas-profile]
+  instances: 1
```

Comment:
OIC, I was reading the cloud-config wrong and looking at the web nodes 😁 

In that case, let's go for 2x `m4.large` ?  It'll be cheaper than  1x `m3.xlarge`